src/Microsoft.Identity.Client/DefaultEvent.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Identity.Client.Internal;
+
+namespace Microsoft.Identity.Client
+{
+    internal class DefaultEvent : EventBase
+    {
+        public DefaultEvent(string clientId) : base((string) (EventBase.EventNamePrefix + "default_event"))
+        {
+            this["client_id"] = clientId;
+            this["sdk_platform"] = PlatformPlugin.PlatformInformation.GetProductName();
+            this["sdk_version"] = MsalIdHelper.GetMsalVersion();
+            // this["application_name"] = MsalIdHelper.GetApplicationName();  // Not yet implemented
```

Comment:
OK I'll add a new comment line

    // TODO: The following implementation will be used after the 3 helpers being implemented (in a separated PR)

BUT I'll still keep the 3 commented lines because they will be the correct implementation.